src/TheRestOfYourLife/color.h

```diff
@@ -35,13 +35,12 @@ void write_color(std::ostream& out, const color& pixel_color, int samples_per_pi
     if (g != g) g = 0.0;
     if (b != b) b = 0.0;
 
-    // Divide the color by the number of samples.
-    auto scale = 1.0 / samples_per_pixel;
-    r *= scale;
-    g *= scale;
-    b *= scale;
+    // Divide the color by the sampling scale.
+    r *= spp_scale;
+    g *= spp_scale;
+    b *= spp_scale;
 
-    // Apply a linear to gamma transform for gamma 2
+    // Apply the linear to gamma transform.
```

Comment:
Doesn't appear that `linear_to_gamma` changes between books, i assume we'd want consistency in code comments? 

Perhaps better addressed in it's own issue.